hack/tools/go.mod

```diff
@@ -1,13 +1,13 @@
 module sigs.k8s.io/cluster-api-provider-azure/hack/tools
 
-go 1.12
+go 1.13
 
 require (
 	github.com/golang/mock v1.3.1
-	github.com/golangci/golangci-lint v1.19.1
+	github.com/golangci/golangci-lint v1.23.3
 	k8s.io/code-generator v0.0.0-20190923155300-6206bfaf5c98
 	sigs.k8s.io/cluster-api/hack/tools v0.0.0-20191002152509-a6b5ba419bad
-	sigs.k8s.io/controller-tools v0.2.1
+	sigs.k8s.io/controller-tools v0.2.4
```

Comment:
0.2.5 is out already, why not use the latest patch?

Reply:
@CecileRobertMichon yes, i will also review the others

Reply:
per @justaugustus comments, we will bump this in other PR. do we need an issue for those things?

Reply:
@cpanato -- It's fine to just issue a follow-up PR with the controller-tools changes.
/hold until those changes are removed from this PR

Reply:
sounds good @justaugustus 
revert back this change